examples/regular_polygons.py

```diff
@@ -0,0 +1,89 @@
+import random
+import sys
+
+import geometry
+import pygame as pg
+
+FPS = 60
+RES = WIDTH, HEIGHT = 640, 480
+DARK_GREY = (10, 10, 10)
+SIDES = [3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+COLORS = {
+    3: (255, 0, 0),
```

Comment:
Instead of having this, I think it would be better to just come up with a random RGB and side values instead of having this list here.

Reply:
The problem with random colors is that sometimes colors are not that good on the screen.